agent-install/edgeNodeFiles.sh

```diff
@@ -150,6 +150,14 @@ function checkPrereqsAndInput () {
     echo " - CLUSTER_USER: $CLUSTER_USER"
     echo " - CLUSTER_PW set"
     echo ""
+
+    echo "Checking the value of parament option..."
+    if [[ -d $DIR ]]; then
+	echo ' - Value of parament options valid'
+    else
+	fatal 1 "ERROR: the value of parament option '-f' isn't an existing directory ..."
+    fi 
+    echo ""
```

Comment:
Did you test this in the case when `-f <dir>` was **not** specified? I think your code would exit with the error message, when it shouldn't. I already told you that you can't submit PRs without thoroughly testing them first.

1. So your check should be `if [[ -n $DIR && ! -d $DIR ]]; then ...`
2. The check should be in the code section that starts with `echo "Checking command arguments ...", because it we don't need a 2nd good output after " - Command arguments valid", and in the error case it would be confusing to say the command arguments are valid and then give an error message about `-f`
3. The word `parament` is misspelled, but also unnecessary

Reply:
Yea, it is my fault. I don't test this included that condition,  as it always check the some tools , such as `oc`, `cloudctl`. 

This time, I simply disable those checking, and do following testing:

1. without `-f` option:
```
% ./edgeNodeFiles.sh x86_64-macOS 
Checking command arguments ...
 - Command arguments valid
```
2. `-f` with non-directory:
```
% ls -ld tmp                            
ls: tmp: No such file or directory
 % ./edgeNodeFiles.sh x86_64-macOS -f tmp
Checking command arguments ...
ERROR: the value of option '-f' isn't an existing directory ...
```
3. `-f` with correct directory:
```
% ls -ld horizon-agent-macos-pkg-x86_64 
drwxr-xr-x@ 4 llhu  staff  128 Jan 11 13:22 horizon-agent-macos-pkg-x86_64
% ./edgeNodeFiles.sh x86_64-macOS -f horizon-agent-macos-pkg-x86_64
Checking command arguments ...
 - Command arguments valid
 ```
@bmpotter ,Could you review this again, I will be careful later. Thanks

Reply:
looks good now, thx